src/components/core/Avatar/Avatar.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from "react";
+
+import { avatarState } from "./Avatar.spec";
+import { Avatar, Button } from "components/core";
+import { Flex, HStack } from "generated/panda/jsx";
+
+import type { Meta, StoryObj } from "@storybook/react";
+
+type Story = StoryObj<typeof Avatar>;
+
+const AvatarToggle = () => {
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  return (
+    <Flex direction="column" gap={2}>
+      {isLoaded ? (
+        <Avatar src="/img/logo.png" alt="avatar" id="" />
+      ) : (
+        <Avatar src="" alt="" id="" />
+      )}
+      <Button
+        data-testid="toggle"
+        variant="ghost"
+        p={2}
+        onClick={() => setIsLoaded(!isLoaded)}
+      >
+        {isLoaded ? "Toggle to fallback" : "Toggle to image"}
+      </Button>
+    </Flex>
+  );
+};
+
+export const Toggle: Story = {
+  render: () => <AvatarToggle />,
+};
+
+export const Variants: Story = {
+  render: () => (
+    <HStack gap={2}>
+      <Avatar src="/img/logo.png" alt="avatar" id="" />
+      <Avatar variant="square" src="/img/logo.png" alt="avatar" id="" />
+    </HStack>
+  ),
+};
+
+export const Sizes: Story = {
+  render: () => (
+    <HStack gap={2}>
+      <Avatar size="xs" src="/img/logo.png" alt="avatar" id="" />
+      <Avatar size="sm" src="/img/logo.png" alt="avatar" id="" />
+      <Avatar src="/img/logo.png" alt="avatar" id="" />
+      <Avatar size="lg" src="/img/logo.png" alt="avatar" id="" />
+      <Avatar size="xl" src="/img/logo.png" alt="avatar" id="" />
+      <Avatar size="2xl" src="/img/logo.png" alt="avatar" id="" />
+      <Avatar size="3xl" src="/img/logo.png" alt="avatar" id="" />
+    </HStack>
+  ),
+};
+
+export const AvatarState: Story = {
+  ...Toggle,
+  play: avatarState,
+  name: "[TEST] Avatar State",
+  tags: ["test"],
+};
+
+// TODO remove explicit type annotation, required due to `pnpm` bug (and therefore Yarn with `pnpm` linker); https://github.com/microsoft/TypeScript/issues/47663
+const meta: Meta<typeof Avatar> = {
```

Comment:
Should be able to remove this now!

Reply:
Yup, I saw my branch was behind again and caught that! Thanks for the heads up.